README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
 
-For ease of local setup, we have created a docker compose setup which builds docker images for all the necessary components. It also uses nginx and mkcert to host the app locally over HTTPS. 
-
-To use this, you'll need to install the following:
-
-1. Install [docker and docker compose](https://docs.docker.com/get-docker/).
-1. Install [mkcert](https://github.com/FiloSottile/mkcert#installation)
-1. Install Node.js 14, either using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) or your preferred method. This is used for yarn scripts and pre-commit hooks.
-1. Install AdoptOpenJDK 11, either using [jabba](https://github.com/shyiko/jabba) or your preferred method. This is useful for IDE integration (IntelliJ, VS Code, etc.)
-1. Install yarn by running `npm install --global yarn`.
-
-Once those dependencies are installed, follow these steps to get the app up and running:
-
-1. Clone this repository: `git clone git@github.com:CDCgov/prime-simplereport.git`
-2. Run `yarn install` in the root of the repository to install pre-commit hooks using lefthook
-3. Run `cp .env.sample .env` in the root of the repository, then obtain the needed secrets from Azure or another developer. Be sure to set the `OKTA_API_KEY` environment variable. You can generate an API token for yourself by logging into the Okta Preview [admin panel](https://hhs-prime-admin.oktapreview.com) and going into Security > API > Tokens.
-4. Run `sudo vim /etc/hosts` (or use your editor of choice) and add a line to the bottom with the following contents and save: `127.0.0.1 localhost.simplereport.gov`
-5. Follow these steps from GitHub for [authenticating to the container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
-6. Run `yarn start` in the root of the repository to start the app
-7. Make your user a support admin by assigning yourself the `SR-DEV-ADMINS` group in Okta Preview.
-8. When the frontend and backend builds are complete, visit https://localhost.simplereport.gov/app
-
-You should see the support admin page. The application does not yet start with any seed data, so you'll need to add your own organization and facilities. You can create your first org at https://localhost.simplereport.gov/app/sign-up.
-
-#### Notes
-Make sure docker has access to enough resources on your local machine. You'll know it doesn't if you get an error from any of your containers similar to this:
-```
-"The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process."
-```
-
-How to update resources limits on your [Mac](https://docs.docker.com/desktop/mac/#resources), and [Windows](https://docs.docker.com/desktop/windows/#resources) machines.
-
-## Running outside of docker
```

Comment:
⚠️ Moved to https://github.com/CDCgov/prime-simplereport/wiki/Running-outside-of-docker